lib/src/sass/tables.scss

```diff
@@ -1,6 +1,8 @@
-@import "./colors.scss";
-@import "./functions.scss";
-@import "../../node_modules/font-awesome/scss/variables"; // get rid of this when we roll our own font
+$fa-font-path: '../../../node_modules/font-awesome/fonts';
```

Comment:
Not a big deal, but I would call this file _tables.scss

Reply:
I removed the _ because I thought the _ meant that you wouldn't import it directly, just indirectly via cashmere.scss. I separated tables.scss out as it imports the other common .scss files itself. It is probably worth discussing it though

Reply:
Im ok with that